rpcs3/Emu/Io/usio.cpp

```diff
@@ -104,7 +104,19 @@ void usb_device_usio::control_transfer(u8 bmRequestType, u8 bRequest, u16 wValue
 	}
 }
 
+enum TaikoButtonMapping : u8
+{
+	TAIKO_BTN_ENTER   = 0x02,
+	TAIKO_BTN_DOWN    = 0x10,
+	TAIKO_BTN_UP      = 0x20,
+	TAIKO_BTN_SERVICE = 0x40
+};
+
 extern bool is_input_allowed();
+bool is_test         = false;
```

Comment:
Please avoid using global variables unless strictly necessary(which it is really not here).
You should put those variables as private members in the class usb_device_usio(in usio.h).

Reply:
Ah, sorry. Changed and pushed. 🙇